ucan-invocation/constants.js

```diff
@@ -7,6 +7,11 @@ import {
   remove as uploadRemove
 } from '@web3-storage/capabilities/upload'
 
+export const CONTENT_TYPE = {
+  WORKFLOW: 'application/invocations+car',
+  RECEIPT: 'application/receipt+dag-cbor'
```

Comment:
- the + suffix denotes an existing media type which are registered as `vnd.ipld.car` and `vnd.ipld.dag-cbor`
- the prefix is a tree, and we can add things to the vnd tree without much fuss like:
```
application/vnd.w3.ucan+vnd.ipld.car
application/vnd.w3.ucan.receipt+vnd.ipld.dag-cbor
```
of if we really don't care then
```
application/x.ucan+vnd.ipld.car
application/x.ucan.receipt+vnd.ipld.dag-cbor
```
see: https://en.wikipedia.org/wiki/Media_type#Naming

Reply:
https://github.com/ipfs/in-web-browsers/issues/201
https://github.com/ipfs/in-web-browsers/issues/202